cni/plugin.go

```diff
@@ -168,6 +169,20 @@ func (plugin *Plugin) GetEndpointID(args *cniSkel.CmdArgs) string {
 	return containerID + "-" + args.IfName
 }
 
+func (plugin *Plugin) GetCNIArgs(args string) map[string]interface{} {
+	argsMap := make(map[string]interface{})
+
+	data := strings.Split(args, ";")
+	for _, pair := range data {
+		items := strings.Split(pair, "=")
+		if len(items) > 1 {
+			argsMap[items[0]] = items[1]
+		}
+	}
+
```

Comment:
Is args always guaranteed to have this format? "key0=val0; key1=val1"

Reply:
yes